content/docs/started/getting-started.md

```diff
@@ -66,11 +66,12 @@ Download, set up, and deploy (If you prefer to work from source code, feel free
 1. Run the following commands to setup and deploy Kubeflow:
     
     ```
-    ${KUBEFLOW_SRC}/scripts/kfctl.sh init ${KFAPP} --platform none
+    ${KUBEFLOW_SRC}/scripts/kfctl.sh init ${KFAPP} --platform none --zone ${ZONE}
     cd ${KFAPP}
     ${KUBEFLOW_SRC}/scripts/kfctl.sh generate k8s
     ${KUBEFLOW_SRC}/scripts/kfctl.sh apply k8s
     ```
+   * **${ZONE}** is the deployment zone, make sure this [zone has GPUs](https://cloud.google.com/compute/docs/regions-zones/).
```

Comment:
Not everyone needs GPUs, so we should note that they should check the availability GPUs in the zone only if they need GPUs for their workload.

Reply:
That's true, but the install script does give an error on inavailability.